packages/plugin/src/browser/plugin-api-frontend-contribution.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from 'inversify';
+import { Command, CommandRegistry, CommandContribution } from '@theia/core/lib/common';
+import { HostedPluginClient } from './hosted-plugin-manager-client';
+
+export namespace HostedPluginCommands {
+    export const RUN: Command = {
+        id: 'hosted-plugin:run',
+        label: 'Hosted Plugin: Run'
```

Comment:
`Start a new instance` ?

Reply:
We cannot start more then one hosted instance from the current IDE (but can do it in a chain). To me Run is better